numeral.js

```diff
@@ -54,10 +54,15 @@
           boundedPrecision = minDecimals;
         }
 
+        // Number.toFixed requires it's argument be between 0 and 20, inclusive.
+        // See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toFixed
+        boundedPrecision = Math.max(Math.min(boundedPrecision, 20), 0);
```

Comment:
I do need to add a test for the optional case though, didn't see that case before.